tests/testthat/test-methods.R

```diff
@@ -170,6 +170,19 @@ local({
   lints <- lint(text = "a", linters = test_linter())
   lint <- lints[[1L]]
 
+  widths <- c(10L, 20L, 40L, 80L)
+  test_names <- paste0(": width = ", widths)
+
+  patrick::with_parameters_test_that(
+    "print.lint, print.lints support optional message wrapping",
+    {
+      withr::local_options(c(lintr.format_width = width))
```

Comment:
I was just trying to avoid another level of nesting:

```r
local({
  patrick::with_parameters_test_that({
    withr::with_options({
      ... 
    })
  })
})
```